### ISSUE TITLE
508-defect-2 [SCREENREADER]: Loading indicator should have aria-live="polite" value

### ISSUE THREAD

**Issue by noahgelman** [department-of-veterans-affairs/va.gov-team#32813]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Noah

## Details

The loading indicator needs to announce to screen readers that the content is loading. Adding an aria-live value to the component will read the message to the user automatically

## Acceptance Criteria

- [ ] Add `aria-live="polite"` to the loading message container. The spinner itself does not need to be inside `aria-live`

**Comment by bkjohnson** [comment]
Solved in the linked PR
